src/libraries/System.IO.FileSystem/src/System/IO/FileStatus.Unix.cs

```diff
@@ -242,7 +242,7 @@ internal DateTimeOffset GetCreationTime(ReadOnlySpan<char> path, bool continueOn
         {
             EnsureCachesInitialized(path, continueOnError);
             if (!_exists)
-                return DateTimeOffset.FromFileTime(0);
+                return new DateTimeOffset(DateTime.FromFileTimeUtc(0));
```

Comment:
Isn't this a breaking change?  Previously the DateTimeOffset that was returned would have had a Kind == Local, but now it's going to have a Kind == Utc.

Reply:
Ignore that... I was reading it as DateTime rather than DateTimeOffset.

Reply:
But... it's still a semantic change... won't the returned DateTimeOffset now have a different value than it did previously, with a different offset?

Reply:
This is called from 2 locations:
 1) [FileSystemEntry.CreationTimeUtc](https://github.com/dotnet/runtime/blob/ebba45155/src/libraries/System.IO.FileSystem/src/System/IO/Enumeration/FileSystemEntry.Unix.cs#L122) which by it's name already expects `DateTimeOffset` to be `Utc`, this is what added unit test ensures.
 2) [FileSystemInfo.CreationTimeCore](https://github.com/dotnet/runtime/blob/ebba45155/src/libraries/System.IO.FileSystem/src/System/IO/FileSystemInfo.Unix.cs#L46) which is only called from [FileSystemInfo.CreationTimeUtc](https://github.com/dotnet/runtime/blob/ebba4515/src/libraries/System.IO.FileSystem/src/System/IO/FileSystemInfo.cs#L81), notice that [FileSystemInfo.CreationTime](https://github.com/dotnet/runtime/blob/ebba4515/src/libraries/System.IO.FileSystem/src/System/IO/FileSystemInfo.cs#L75) calls [FileSystemInfo.CreationTimeUtc.ToLocalTime()](https://github.com/dotnet/runtime/blob/ebba4515/src/libraries/System.IO.FileSystem/src/System/IO/FileSystemInfo.cs#L81)

I assumed [FileSystemInfo.CreationTimeCore](https://github.com/dotnet/runtime/blob/01b7e73cd3/src/libraries/System.IO.FileSystem/src/System/IO/FileSystemInfo.Windows.cs#L97) on Windows also returns UTC, but while investigating for this answer I noticed that it calls [FILE_TIME.ToDateTimeOffset](https://github.com/dotnet/runtime/blob/01b7e73cd/src/libraries/Common/src/Interop/Windows/Kernel32/Interop.FILE_TIME.cs#L23) which is also performing `ToLocalTime` which I think is as unnecessary on Windows as on Unix.

I will change Windows as well and write Performance test as requested by @adamsitnik.